fronts-client/src/actions/Editions.tsx

```diff
@@ -78,6 +65,50 @@ export const publishEditionIssue = (
   }
 };
 
+export const publishEditionIssue = (
+  id: string,
+  version?: string
+): ThunkResult<Promise<void>> => async (dispatch: Dispatch) => {
+  if (!version) {
+    alert('No issue version found');
```

Comment:
This suggests we need to have completed a round of proofing before we can publish? Should we disable the publish button if there's no proofed version? Or should we expand this message to describe a way to resolve this?

Reply:
Good idea. Message expanded with instructions.